bgcval2/analysis_compare.py

```diff
@@ -140,12 +140,52 @@ def apply_shifttimes(mdata, jobID, shifttimes):
     return times, datas
 
 
+def apply_timerange(times, datas, jobID, timeranges):
+    """
+    This version takes the times and dataafter apply_shifttimes ,
+    and removes things outside the time range requested.
+    the value of the range is provided in the yaml file.
+
+    Outputs two lists: dates & data.
+    """
+    if 0 in [len(times), len(datas), ]:
+       return times, datas
+
+    print('apply_timerange', jobID, timeranges)
+
+    timerange = timeranges.get(jobID, None)
+
+    print('apply_timerange', timerange)    
+    if timerange is None: 
+       print('apply_timerange', timerange, 'is', None) 
```

Comment:
```suggestion
       print('apply_timerange: timerange is', None) 
```

Reply:
how it is now would print "apply_timerange None is None" (famous quote by Teresa May)